### ISSUE TITLE
[Housekeeping] Investigate use of data-element in docs

### ISSUE THREAD

**Issue by RasmusKjeldgaard** [kirbydesign/designsystem#3772]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the housekeeping request
In the cookbook, mainly on the item showcase, we use `<data>` liberally. I do think it is wrong use of the element, as it should have a `value` attribute with a machine-readable version of the content in the element. 

In a lot of the examples, the data is already machine-readable, and adds no value.

In addition, the missing value appears in automated tests as a accessibility issue for all consuming teams copying our approach.

## Are there any additional context?

[MDN Docs for data](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/data)

**Comment by RasmusKjeldgaard** [comment]
Related: #3847
Maybe we should remove the style overrides provided by item from the data element as suggested by @troelslenda, so the data-element is still usable when the use-case actually merits it?

And then make one good use-case showcasing proper use of `<data>`, while removing it from the rest of the examples.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
